bazel/repositories.bzl

```diff
@@ -16,3 +18,10 @@ def mod_pagespeed_dependencies():
         url = "https://github.com/google/brotli/archive/%s.tar.gz" % BROTLI_COMMIT,
     )
 
+    http_archive(
+        name = "zlib",
```

Comment:
I think bazel has a convention to name these, but I don't know from the top of my head. Renaming this also disambiguates with the `cc_library` rule below. Maybe add a TODO so we can do this later?

Reply:
added TODO to rename library as per bazel naming conventions.
As of now, I have used name **"zlib"** since building cc_library directly generates **libzlib.a** library (file with same name getting generated from gyp build of third party libraries)

Reply:
I think the `.a` name is determined below via 

```
cc_library(
    name = "zlib",
```

regardless, lgtm as-is, thanks for adding the todo